docs/README.md

```diff
@@ -20,8 +20,8 @@ These actions can be executed separately or combined sequentially.
 
 # Requirements
 
--   XL Deploy versions supported: from 9.5.1 to 10.1.0
--   Jenkins LTS versions supported: from 2.60.1 to 2.277.2
+-   Digital.ai Deploy versions supported: from 9.7.x to 10.3.x
```

Comment:
Instead of ' Digital.ai Deploy versions supported: from 9.7.x to 10.3.x'
use 
 [A supported version of DAI Deploy.](https://support.digital.ai/hc/en-us/articles/360016879780-XebiaLabs-Supported-Product-Versions#support-overview-0-3)